.github/workflows/unit_tests.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
         pytype --jobs auto --disable=import-error,module-attr jetstream/ benchmarks/
     - name: Analysing the code with pylint
       run: |
-        pylint jetstream/ benchmarks/
+        make check
```

Comment:
Awesome! Do you mind also replacing other commands in this CI? Thanks!

Reply:
Sure!

Reply:
Done